xmtp_mls/migrations/2024-04-08-180113_group_added_by_address/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE groups
+ADD COLUMN added_by_address TEXT
```

Comment:
@zombieobject I think @richardhuaaa was suggesting that this is NOT NULL do you mind doing that in a follow up PR?